z_infra_provisioning/cloud_infra_center/ocp_upi/README.md

```diff
@@ -348,6 +417,8 @@ The playbook places the Control Plane in a Server Group with "soft anti-affinity
 
 The master nodes should load the initial Ignition and then keep waiting until the bootstrap node stands up the Machine Config Server which will provide the rest of the configuration.
 
+***Note***: This playbook will be failed with `UserWarning: You are specifying a cacert for the cloud envvars:RegionOne but also to ignore the host verification`. You can monitor this issue [#6541](https://github.ibm.com/zvc/planning/issues/6541) to get latest status.
```

Comment:
The issue link is internal and cannot be opened with customers.

Reply:
yes, I don't know how to public this issue. Maybe we can talk it at the meeting.

Reply:
"will be failed" should be "will fail"

Reply:
As far as having a GHE (Github Enterprise) issue be made publicly or mirrored, I don't know of a way. You could list out your known issues some place, a new Markdown file or in the README. You could also open a Git issue in this repository and past the same content into so you have a public copy of the git issue. It is redundant and causes you to track changes in 2 places.

Reply:
Yes, good idea, just add an issue in this repository.